operator/src/main/resources/scripts/utils.sh

```diff
@@ -889,3 +889,94 @@ function adjustPath() {
+#            See also 'commonMount.sh'.
+#
+function checkCommonMount() {
+  # check commonMount results (if any)
+  if [ -z "$COMMON_MOUNT_PATHS" ]; then
+    trace FINE "Common Mount: Skipping common mount checks (no common mounts configured)."
+    return
+  fi
+
+  trace FINE "Common Mount: COMMON_MOUNT_PATHS is '$COMMON_MOUNT_PATHS'."
+  for COMMON_MOUNT_PATH in ${COMMON_MOUNT_PATHS/,/ }; do
+    trace FINE "Common Mount: COMMON_MOUNT_PATH is '$COMMON_MOUNT_PATH'."
+    traceDirs $COMMON_MOUNT_PATH
+    touch ${COMMON_MOUNT_PATH}/testaccess.tmp
+    if [ $? -ne 0 ]; then
+      trace SEVERE "Common Mount: Cannot write to the COMMON_MOUNT_PATH '${COMMON_MOUNT_PATH}'. " \
+                   "This path is configurable using the domain resource 'spec.commonMountVolumes.mountPath' " \
+                   "attribute." && return 1
+    fi
+    rm -f ${COMMON_MOUNT_PATH}/testaccess.tmp || return 1
+
+    out_files=$(set -o pipefail ; ls -1 $COMMON_MOUNT_PATH/commonMountLogs/*.out 2>1 | sort --version-sort) \
+      || (trace SEVERE "Common Mount: Assertion failure. No files found in '$COMMON_MOUNT_PATH/commonMountLogs/*.out" \
+      && return 1)
+    severe_found=false
+    for out_file in $out_files; do
+      if [ "$(grep -c SEVERE $out_file)" != "0" ]; then
+        trace SEVERE "Common Mount: Error found in file '${out_file}' while initializing commonMount."
+        severe_found=true
+      elif [ "$(grep -c successfully $out_file)" = "0" ]; then
+        trace SEVERE "Common Mount: Command execution was unsuccessful in file '${out_file}' while initializing commonMount."
+        severe_found=true
+      fi
+      trace "Common Mount: Contents of '${out_file}':"
+      cat $out_file
+      trace "Common Mount: End of '${out_file}' contents"
+    done
+    [ "${severe_found}" = "true" ] && return 1
+    rm -fr $COMMON_MOUNT_PATH/commonMountLogs
+    [ -z "$(ls -A $COMMON_MOUNT_PATH)" ] \
+      && trace SEVERE "Common Mount: No files found in '$COMMON_MOUNT_PATH'. " \
+       "Do your commonMount images have files in their '$COMMON_MOUNT_PATH' directories? " \
+       "This path is configurable using the domain resource 'spec.commonMountVolumes.mountPath' attribute." \
+      && return 1
+  done
+  return 0
+}
+
+#
+# initCommonMount
+#   purpose: Execute the COMMON_MOUNT_COMMAND specified as part of the common mount init container.
+#            If the specified COMMON_MOUNT_COMMAND is empty, it logs an error message and returns.
+#            If the COMMON_MOUNT_PATH directory doesn't exist or is empty, it logs error and returns.
+#            If the command execution fails, it logs errror message with failure details. Otherwise it
+#            logs a success message with details.
+#            See also 'commonMount.sh'.
+#
+function initCommonMount() {
+
+  if [ -z "${COMMON_MOUNT_COMMAND}" ]; then
+    trace ERROR "Common Mount: The 'serverPod.commonMounts.command' is empty for the " \
```

Comment:
Minor:  it's more clear to use 'trace SEVERE' instead of 'trace ERROR'  (it's more clear as 'trace ERROR' in this location secretly depends on the fact that trace implicitly converts an ERROR to SEVERE, the operator greps for SEVERE, and the checkCommonMount script greps for it too)